### ISSUE TITLE
A11y_AzureDataStudio_JupyterBook_ScreenReader : While Voiceover is on ,unable to navigate the tree items.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7809]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."

GitHubTags:#A11y_ADS_OctTestPass;#A11yMAS;#A11yTCS;#MAC1.3.1;#DesktopApp;#VoiceOver;#SQL AzureDataStudio;#Mac

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS:macOS Mojave Version 10.14.6(18G87)

**Additional Details:**
Tool Name : VoiceOver

**Steps to Reproduce:**
1.Install and Launch the application the azure data studio application.
2.Connet to the servers with the required credentials.
3.Navigate to the control panel and click on the "jupyter books" icon.
4.Press CRTL+SHIFT+P and search "jupyter books: SQL Server 2019 Guide"
5."Jupyter Books" should open.
6.Start Voiceover.
7.Navigate till “Saved book” using tab.
8. Observe that unable to navigate the tree items.

**Actual:**
While screen reader is ON, unable to navigate the tree items using arrows navigation as well as with the ctrl-option-arrows.

**Expected:**
While screen reader is ON ,should be able to navigate the tree items using arrows.

**User Impact:**
Screen reader user will find it difficult to navigate the screen .

**Attachment for Reference:**
[7809_A11y_AzureDataStudio_JupyterBook_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3742799/7809_A11y_AzureDataStudio_JupyterBook_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chlafreniere** [comment]
This bug also reproes in vs code with the files viewlet.

**Comment by chlafreniere** [comment]
I have attempted to create an issue on the VS Code side for this, but am unable to do so, as they've temporarily stopped non-collaborators from being able to create new issues. 

While having Voice Over enabled on Mac, cannot navigate tree items with the keyboard.

As an example, trying to use the arrow keys to navigate the File Explorer results in Voice Over reading one character at a time.

Are there any workarounds for this issue currently?

**Comment by chlafreniere** [comment]
Issue tracked in VS Code in https://github.com/microsoft/vscode/issues/88181

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
![image](https://user-images.githubusercontent.com/54656883/72051841-01abe500-32ea-11ea-9cbf-41486420d2f6.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_Mac;

**Comment by Rama8345** [comment]
A11ySev2

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
